kit/src/components/message/mod.rs

```diff
@@ -68,6 +68,8 @@ pub struct Props<'a> {
     // An optional field that, if set, will be used as the text content of a nested p element with a class of "text".
     with_text: Option<String>,
 
+    with_text_to_render: Option<String>,
```

Comment:
I dislike this name. it's basically the same as `with_text`. I feel that the name doesn't adequately describe what the code does. 

after looking at how this is used in `render_message()`, I believe this variable is unnecessary.

Reply:
ok I understand now. I can't think of a better way to do this at the moment though it would be nice if we could avoid storing messages in an intermediate form. For now i'll request it be renamed to `tagged_text` because that's what it is.